### DISCUSSION TITLE
Github Actions is not updating

### DISCUSSION THREAD

**Discussion by JamesVanWaza** [community/community#25279] [Actions]
<p>I have created a Github action but every time I keep running it shows the old version. This is the error I keep getting. See screenshot attached. Since this is my first post, I can’t put more than one screenshot, let me know the way forward.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/f/efbf349e1fc2d660c348e5bd31ab9b92ecf9e0ba.jpeg" data-download-href="/uploads/short-url/ycTLPjfmPHUBy2xosRbDNBPshFo.jpeg?dl=1" title="Screen Shot 2022-01-31 at 7.28.39 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/4000076/181090051-89826dc4-2c28-4c48-ba38-4755d6e09834.jpeg" alt="Screen Shot 2022-01-31 at 7.28.39 PM" data-base62-sha1="ycTLPjfmPHUBy2xosRbDNBPshFo" width="690" height="370" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/f/efbf349e1fc2d660c348e5bd31ab9b92ecf9e0ba_2_690x370.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/e/f/efbf349e1fc2d660c348e5bd31ab9b92ecf9e0ba_2_1035x555.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/f/efbf349e1fc2d660c348e5bd31ab9b92ecf9e0ba_2_1380x740.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/f/efbf349e1fc2d660c348e5bd31ab9b92ecf9e0ba_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2022-01-31 at 7.28.39 PM</span><span class="informations">1920×1032 119 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<pre><code class="lang-auto">![Screen Shot 2022-01-31 at 7.28.51 PM|690x246](upload://aYjAWRhCfcNv9rvIx9AShtaXxDH.jpeg)
![Screen Shot 2022-01-31 at 7.29.07 PM|690x194](upload://62GI1Bk8oxiUrX0elTJmOWT60Y0.png)
![Screen Shot 2022-01-31 at 7.29.31 PM|690x227](upload://pQYNs6ss6UVOUYVBpudgS0uz2Wu.png)
![Screen Shot 2022-01-31 at 7.29.42 PM|690x61](upload://uDGmmwP1ao5iyduqBanArOEvk4i.png)
</code></pre>
<p>I don’t know why its not updating on Github Actions, yet I have done all the steps required.</p>

**Comment by airtower-luna** [comment]
<aside class="quote no-group" data-username="JamesVanWaza" data-post="1" data-topic="226646">
<div class="title">
<div class="quote-controls"></div>
 JamesVanWaza:</div>
<blockquote>
<p>I have created a Github action but every time I keep running it shows the old version.</p>
</blockquote>
</aside>
<p>In which context does this happen? Like, <em>how</em> do you run it, and the old version of <em>what</em> is used? Which version of it should be used?</p>
<p>It’s best if you can link a workflow log.</p>

**Comment by JamesVanWaza** [comment]
<p>Hi <a class="mention" href="/airtower-luna">@airtower-luna</a> this is happening when I run the Github action on the cli so the project is deploying to Firebase every time I push a commit. The error I keep getting is the target is not the correct one yet, I have already updated it multiple times. is there a way I can send the screenshots so you can review them?</p>

**Comment by JamesVanWaza** [comment, marked as answer]
<p>The solution is deleting the target in <code>firebase.json</code> delete <code>firebase-hosting-merge.yml</code> and <code>firebase-hosting-pull-request.yml</code> and rerunning <code>firebase init</code></p>
